### ISSUE TITLE
Support for X360 S;G?

### ISSUE THREAD

**Issue by dzastsed** [CommitteeOfZero/sc3tools#5]
is there any chance you could add support for xbox 360 version of steins;gate script files? It seems that the decompression _partially_ does work already with some scripts:
![image](https://user-images.githubusercontent.com/63074962/172345625-6e11352a-74b5-48ce-acda-e75dac2bbb0f.png)
however, resulting text files are just gibberish for the most part (it seems that line ends do get properly decoded though?):
![image](https://user-images.githubusercontent.com/63074962/172345718-01a79e7a-4673-468e-a4fe-809eabf62539.png)
some script files also throw out this error:
![image](https://user-images.githubusercontent.com/63074962/172345797-28e54d7a-86a3-4a8a-aba1-d7c6ca363773.png)
and results in gibberish as well (apart from different text colors and line ends):
![image](https://user-images.githubusercontent.com/63074962/172346053-c2697556-d902-4ba6-b485-2bf5b5f12f83.png)
attaching a couple of sample script files too for reference:
[script.zip](https://github.com/CommitteeOfZero/sc3tools/files/8851940/script.zip)

**Comment by dzastsed** [comment]
update: just noticed that I used older build which did not have s;g support yet, so tried extracting with `sghd` argument, but still no luck it seems
![image](https://user-images.githubusercontent.com/63074962/172351537-ed906edb-44eb-4855-94f4-ddb94bde792a.png)
charset changed a bit but still results in gibberish

**Comment by JimStravolski** [comment]
Hello, not sure if it's still unresolved, but you could create a charset file from this image. 

[00009.bmp](https://github.com/user-attachments/files/22930179/00009.bmp)

A partial, incomplete version can be found here:

[sg360char.utf8.txt](https://github.com/user-attachments/files/22930210/sg360char.utf8.txt)

**Comment by dzastsed** [comment]
yeah, still unresolved, honestly i lost interest in doing anything with this
